listenbrainz/spark/handlers.py

```diff
@@ -197,7 +196,7 @@ def handle_dump_imported(data):
     dump_name = data['imported_dump']
     import_completion_time = data['time']
     send_mail(
-        subject='A full data dump has been imported into the Spark cluster',
+        subject='A data dump has been imported into the Spark cluster',
```

Comment:
can we put in info on what type of dump this was in the subject as well? it's helpful when searching through the inbox.